drivers/src/net/realtek/rtl8211f.rs

```diff
@@ -1200,7 +1200,7 @@ where
                     (*desc).desc1 |= 0b11 << 27;
                     desc = desc.add(1);
                 }
-                count = count + 1;
+                count += 1;
```

Comment:
No need to modify the hardware net driver.

Reply:
Ok, I was formatting the code and found a bunch of warnings. Made the changes as he suggested in order to remove the warnings.